src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Workflow\Dumper\GraphvizDumper;
+
+/**
+ * A console command for retrieving information about services.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class WorkflowDumpCommand extends ContainerAwareCommand
+{
+    /**
```

Comment:
can you implement `isEnabled` command to return `false` if you don't add any workflows declared ?